front-end/src/app/shared/components/inputs/additional-info-input/additional-info-input.component.ts

```diff
@@ -1,13 +1,30 @@
-import { Component, Input } from '@angular/core';
+import { Component, OnInit, Input } from '@angular/core';
 import { BaseInputComponent } from '../base-input.component';
 
 @Component({
   selector: 'app-additional-info-input',
   templateUrl: './additional-info-input.component.html',
   styleUrls: ['./additional-info-input.component.scss'],
 })
-export class AdditionalInfoInputComponent extends BaseInputComponent {
+export class AdditionalInfoInputComponent extends BaseInputComponent implements OnInit {
   @Input() descriptionIsSystemGenerated = false;
   @Input() contributionPurposeDescriptionLabel = '';
   @Input() purposeDescriptionLabelNotice?: string;
+  @Input() purposeFieldTitle?: string; //Specifies a specific title for the CPD field
+  @Input() schedule?: 'A' | 'B'; //Specifies a schedule so that a default CPD title can be used
```

Comment:
This will depend heavily on how @mjtravers decides to implement schedule B.  consider that the form field for the purpose description is `contribution_purpose_descrip`, a ScheduleA-only field.  Matt might go one route, which is to use a different component for `expenditure_purpose_descrip`, in that scenario, this component would never be used for ScheduleB.  More likely, he'll make the component use a different form field depending on the schedule.  If that's the case, you'll want your condition to depend on how he signals the schedule, so we don't have two different data points describing what schedule the component is used for.